src/app_service/service/accounts/service.nim

```diff
@@ -677,8 +434,9 @@ QtObject:
     debug "account logged in"
     self.setLocalAccountSettingsFile()
 
-  proc login*(self: Service, account: AccountDto, hashedPassword: string) =
+  proc login*(self: Service, account: AccountDto, hashedPassword: string, chatPrivateKey: string = "", mnemonic: string = "") =
     try:
+      # WARNING: Is this keystor migration still needed?
```

Comment:
I'm not sure what this one is

Reply:
No idea too. Opened a separate issue to check this: https://github.com/status-im/status-desktop/issues/15188